source/structured-streaming.txt

```diff
@@ -160,6 +213,74 @@ more about continuous processing, see the `Spark documentation <https://spark.ap
 
    tabs:
 
+     - id: java-sync
+       content: |
+
+         To use continuous processing with the MongoDB Spark Connector, 
+         add the ``trigger()`` method to the ``writeStream`` property 
+         of the streaming Dataset or DataFrame that you create from 
+         your MongoDB read stream. In your ``trigger()``, use the 
+         ``Trigger.Continuous()`` property.
+         
+         .. note:: 
+         
+            The connector populates its read stream from your MongoDB 
+            deployment's change stream. To populate your change stream, 
```

Comment:
No, it's not official in any way - I thought it was descriptive, but I can see how it might be strange for some. 

How about:
"The connector reads from your MongoDB deployment's change stream. To push updates to the change stream, perform update operations on your database."

Reply:
Also turned this into an include file